app/portainer/views/account/create-user-access-token/create-user-access-token.html

```diff
@@ -0,0 +1,15 @@
+<rd-header>
+  <rd-header-title title-text="Create access token"></rd-header-title>
+  <rd-header-content> <a ui-sref="portainer.account">User settings</a> &gt; Add access token </rd-header-content>
+</rd-header>
+
+<div class="row">
+  <div class="col-sm-12">
+    <rd-widget>
+      <rd-widget-body>
```

Comment:
we have a widget component, if you want to move this into your react component

Reply:
doesnt provide the right styling :(

![image](https://user-images.githubusercontent.com/63374656/143135010-200e128c-b1fb-4748-9246-3b55af638766.png)

Reply:
exactly what I meant